java/ql/src/experimental/Security/CWE/CWE-352/JsonpInjectionLib.qll

```diff
@@ -77,16 +77,26 @@ class JsonpBuilderExpr extends AddExpr {
   Expr getJsonExpr() { result = this.getLeftOperand().(AddExpr).getRightOperand() }
 }
 
-/** A data flow configuration tracing flow from remote sources to jsonp function name. */
-module RemoteFlowConfig implements DataFlow::ConfigSig {
-  predicate isSource(DataFlow::Node src) { src instanceof RemoteFlowSource }
+/** A data flow configuration tracing flow from threat model sources to jsonp function name. */
+module ThreatModelFlowConfig implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node src) { src instanceof ThreatModelFlowSource }
 
   predicate isSink(DataFlow::Node sink) {
     exists(JsonpBuilderExpr jhe | jhe.getFunctionName() = sink.asExpr())
   }
 }
 
-module RemoteFlow = DataFlow::Global<RemoteFlowConfig>;
+/**
+ * DEPRECATED: Use `ThreatModelFlowConfig` instead.
+ */
+deprecated module RemoteFlowConfig = ThreatModelFlowConfig;
+
+module ThreatModelFlow = DataFlow::Global<ThreatModelFlowConfig>;
+
+/**
+ * DEPRECATED: Use `ThreatModelFlow` instead.
+ */
+deprecated module RemoteFlow = ThreatModelFlow;
```

Comment:
This library is in `experimental` (and the path is non-importable anyway), so I'd say there's no need to deprecate these modules.